src/particles/elements/Elemlist.H

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Axel Huebl, Ji Qiang
+ *
+ * This file is part of ImpactX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_ELEMENTS_LIST_H
+#define IMPACTX_ELEMENTS_LIST_H
+
+#include "Drift.H"
+#include "Sbend.H"
+#include "Quad.H"
+
+#include <variant>
+
+
+namespace impactx
+{
+    using KnownElements = std::variant<Drift, Sbend, Quad>;
```

Comment:
This file is probably added accidentally, we have this already in development :(